components/Button.js

```diff
@@ -70,10 +70,11 @@ class Button extends PureComponent {
       button: true,
       [`button--${color}`]: !!color,
       [`button--${size}`]: !!size,
+      'button--has-icon': !!icon,
     });
 
     return (
-      <button className={buttonClassNames} onClick={onClick} {...rest}>
+      <button type="button" className={buttonClassNames} onClick={onClick} {...rest}>
```

Comment:
Implementing this as a `defaultProp` might be better?

Reply:
Good call.